node/cloud.go

```diff
@@ -50,26 +50,30 @@ type CloudParams struct {
 }
 
 type AWSConfig struct {
-	// AWS profile to use for the node
+	// AWSProfile is the AWS profile in AWS credentials file to use for the node
+	// For more information about AWS Profile, head to https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html#cli-configure-files-format-profile
 	AWSProfile string
 
-	// AWS KeyPair to use for the node
+	// AWSKeyPair is name of the KeyPair used to access the node
 	AWSKeyPair string
 
-	// AWS volume size in GB
+	// AWSVolumeSize is AWS volume size in GB
 	AWSVolumeSize int
 
-	// AWS volume type
+	// AWSVolumeType is the AWS volume type
 	AWSVolumeType string
 
-	// AWS volume IOPS
+	// AWSVolumeIOPS AWS volume IOPS
 	AWSVolumeIOPS int
 
-	// AWS volume throughput
+	// AWSVolumeThroughput is AWS volume throughput
 	AWSVolumeThroughput int
 
```

Comment:
I am not sure if it's a good idea as name and id is 1 to 1 relation.
What happens if they provide sgID and incorrect name ?